.github/workflows/apps/go.mod

```diff
@@ -18,7 +18,7 @@ require (
 	github.com/DataDog/datadog-agent/pkg/util/scrubber v0.64.0-rc.1 // indirect
 	github.com/DataDog/datadog-agent/pkg/version v0.64.0-rc.1 // indirect
 	github.com/DataDog/datadog-go/v5 v5.6.0 // indirect
-	github.com/DataDog/go-libddwaf/v3 v3.5.4 // indirect
+	github.com/DataDog/go-libddwaf/v4 v4.0.0-rc.1 // indirect
```

Comment:
@RomainMuller Is the `rc.1` the released version or should we wait to `v4.0.0` to be tagged to merge?

Reply:
@eliottness wants to make 1 more "breaking" change on the V4 API here, so we _probably_ should hold off on merging this util we cut the `v4.0.0` final...

I think the best way to do this "safely" is to:
- Have the further changes integrated by a PR on top of this one
- Once it is approved, merge this one; then the subsequent one back-to-back

So we keep the change-sets nice and tidy, and "independently reversible"?